login/service_whitebox_test.go

```diff
@@ -0,0 +1,151 @@
+package login
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/almighty/almighty-core/account"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/almighty/almighty-core/token"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/github"
+)
+
+var db *gorm.DB
+var loginService *gitHubOAuth
+
+// Github doesn't allow commiting actual tokens no matter how
+// less privleges the token has.
```

Comment:
Good question! Thank you for taking out time.

Even if the password is changed, the token remains active. This token doesnt have an expiry and can only be invalidated by me from the "Peronsal Tokens" control panel in my Github account.